overlays/bootstrap.nix

```diff
@@ -134,11 +134,11 @@ in {
                 ++ fromUntil "8.10" "8.10.2"   ./patches/ghc/67738db10010fd28a8e997b5c8f83ea591b88a0e.patch
                 ++ final.lib.optional (versionAtLeast "8.6.4" && versionLessThan "8.8") ./patches/ghc/ghc-no-system-linker.patch
 
-                ++ fromUntil "8.10.2" "8.12"   ./patches/ghc/MR3714-backported-to-8.10.2.patch
+                #++ fromUntil "8.10.2" "8.12"   ./patches/ghc/MR3714-backported-to-8.10.2.patch
```

Comment:
We could have probably set upper bound here to 8.10.3 and change `ghc-patches = ghc-patches "8.10.2";` to `ghc-patches = ghc-patches "8.10.3";`